### ISSUE TITLE
bundix -l not working

### ISSUE THREAD

**Issue by aanderse** [manveru/bundix#52]
I have recently updated from bundix 2.3.1 to 2.4.1. Prior to this update I would normally run `bundix -l` in a directory containing a Gemfile and this would create a Gemfile.lock and gemset.nix file, but since the upgrade I get the following error:
```
[nix-shell:~/nixpkgs/pkgs/applications/version-management/redmine]$ bundix --lockfile=Gemfile.lock -l
Traceback (most recent call last):
        8: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/bin/.bundix-wrapped:18:in `<main>'
        7: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/bin/.bundix-wrapped:18:in `load'
        6: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/bin/bundix:5:in `<top (required)>'
        5: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/lib/bundix/commandline.rb:22:in `run'
        4: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/lib/bundix/commandline.rb:35:in `run'
        3: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/lib/bundix/commandline.rb:148:in `build_gemset'
        2: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/lib/bundix.rb:47:in `convert'
        1: from /nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/lib/bundix.rb:177:in `parse_lockfile'
/nix/store/5s5847yw3zmnfyqq6k9f9jd9jylq998s-bundix-2.4.1/lib/ruby/gems/2.5.0/gems/bundix-2.4.1/lib/bundix.rb:177:in `read': No such file or directory @ rb_sysopen - /home/aaron/nixpkgs/pkgs/applications/version-management/redmine/Gemfile.lock (Errno::ENOENT)
```

**Comment by alyssais** [comment]
> 7b3c876a2d20c61ba895002fca1789b3dd11b582 is the first bad commit

**Comment by alyssais** [comment]
Dupe of https://github.com/manveru/bundix/issues/44.

I guess I should make a release.

**Comment by alyssais** [comment]
Released 2.4.2.
